.gitattributes

```diff
@@ -0,0 +1,2 @@
+# This ensures text files use `LF` line endings for consistency, especially for Windows VSCode users.
+* text=auto eol=lf
```

Comment:
Please remove this file

Reply:
Removed.

My VSCode runs on Windows.
The files are converted to CRLF line endings when I clone them on Windows without `.gitattributes`.
The following three files cannot be run in the Linux development container of VSCode:

- .devcontainer/on-create-command.sh
- .devcontainer/post-create-command.sh
- .devcontainer/update-content-command.sh

The file format can be seen in the bottom right of VSCode as shown below:
![image](https://github.com/ros-planning/navigation2/assets/7146994/39d9a7f4-2823-437a-8ef2-fa06a5da2782)


If needed, I can add this setting in the future.